### ISSUE TITLE
Add command to perform actions in cmd+k console

### ISSUE THREAD

**Issue by markmhendrickson** [blockstack/blockstack-app#3400]
This is sorely needed to help with debugging cases where the user doesn't see their ordinals in the wallet, though now the wallet is showing them a new address (regenerated), and so it's not clear how we even confirm its the same wallet (or which are all the taproot addresses they've used to date for depositing inscriptions)

cc @314159265359879

**Comment by markmhendrickson** [comment]
Good use case for https://kbar.vercel.app/

**Comment by mica000** [comment]
Reference
<img width="515" alt="Screenshot 2023-05-15 at 16 34 09" src="https://github.com/hirosystems/wallet/assets/119523541/a52bd28e-d7db-44df-ab1f-f56e10b44739">

**Comment by mica000** [comment]
https://github.com/hirosystems/wallet/assets/119523541/28827653-3674-41a2-8631-be4afe4b86ad

https://www.figma.com/file/Li7qK8ZIG9c5dKSNOPO4iCtv/%E2%9D%96-Design-System?type=design&node-id=4724-42806&t=UEE4bKbRbBIYxTyc-4

**Comment by markmhendrickson** [comment]
Presumably the above design can be a lightweight "skin" of k-bar essentially? 

<img width="686" alt="Screenshot 2023-05-30 at 12 48 29" src="https://github.com/hirosystems/wallet/assets/28991/074b5971-2776-4c59-9923-b18954deadd3">

**Comment by markmhendrickson** [comment]
Also, for the command's output (of addresses), is the idea that we display them in a separate / normal wallet page as a result of selecting the relevant command from k-bar? Instead of showing them in the k-bar UI itself?

**Comment by mica000** [comment]
@markmhx  Not sure I understand what would a "command's output (of addresses)" is?

**Comment by markmhendrickson** [comment]
This issue's particular command would be something like "List all Taproot addresses" and would result in the user viewing a list of all such addresses in their active account. That list would have to be displayed either in the k-bar itself or in a regular view / page, which is otherwise not navigable from anywhere but the k-bar.

**Comment by 314159265359879** [comment]
"all taproot addresses", isn't that number virtually endless? ;-) Just showing the addresses with activity seems a bit to restrictive too. If we want to have a reasonable idea a certain address isn't there for example. 

Lets have an option to reveal the first 10 Taproot addresses for an account, highlight the ones with activity (=ordinals on them). Then reveal and highlight additional ones if there is activity detected on them? I think that will be helpful with troubleshooting.

m/86'/0'/`<account>'/0/0 upto m/86'/0'/`<account>'/0/9 on Sparrow would look like this (account index 0-9):
![image](https://github.com/hirosystems/wallet/assets/33360391/eec6411e-fd69-462e-aaf9-870fb36c8e63)

**Comment by mica000** [comment]
@314159265359879 Could explain why "Just showing the addresses with activity seems a bit to restrictive too."?

Would having a cta with "generate new address" work? then when clicking on it the address is added to the list;

**Comment by 314159265359879** [comment]
@mica000 yes that is a good idea. Lets call it "Reveal another address" (it is much more like revealing, then generating/creating). 

likewise could there be a call to action to reveal the derivation path for these addresses? which adds a column with them?
Address__ | Derivation path
bc1p...3o3 | m/86'/0'/99'/0/0
bc1p...ew2 | m/86'/0'/99'/0/1

**Comment by markmhendrickson** [comment]
How about we show the first 50 Taproot addresses (regardless of whether they have activity) then have "Reveal more addresses" reveal 50 more each time it's selected?

That way users can see a bunch upfront / each time without having to click a bunch, and they can see both ones with and without activity to help diagnose various problems that can arise (related to gaps in which some addresses don't have activity before others do, for example).

And if we're going with a table approach, I'd suggest we also include an index column.
